### DISCUSSION TITLE
Zero config + 404

### DISCUSSION THREAD

**Discussion by craigruks** [vercel/vercel#4052] [General] [answered]
Hi, I have a static site where I utilize zero config + the `cleanUrls` parameter.

I would like to add a route that returns an html page for any HTTP 404 response. What is the best practice to accomplish this?

Thanks!

**Comment by ckipp01** [comment]
Came here to ask this same question. This would be a great feature to have without having to specify all your routes.

**Comment by styfle** [comment, marked as answer]
This is not possible yet. You would need to use `routes`.

https://github.com/zeit/now/issues/3491#issuecomment-580269459

**Reply by ckipp01** [reply]
Thanks for the reply. Any idea on a time-frame for when to expect this? In another support channel I was told
> Regarding custom 404, this will be released in the next couple of weeks.

And that was a month and a half ago.

**Reply by styfle** [reply]
No ETA yet.

Maybe the other support channel was referring to Next.js?

https://nextjs.org/docs/advanced-features/custom-error-page

**Reply by ckipp01** [reply]
Alright, sounds good. Thanks. No, it was pretty clearly about plain static deployments. It was a conversation on Twitter with someone on your support team.

> We deprecated the "builds" and "routes" section. The first was replaced with https://zeit.co/blog/advanced-project-settings and the second by "rewrites", "headers", and "redirects".
Regarding custom 404, this will be released in the next couple of weeks. If you want to use routes, let me know so I can paste an example here.

**Reply by craigruks** [reply]
Thanks for the response @styfle , I'll use routes for now and keep an eye out for upgrades!
